src/plots/plot.ts

```diff
@@ -80,6 +95,7 @@ export class Plot extends Component {
     super._setup();
     this._renderArea = this.content().append("g").classed("render-area", true);
     this.datasets().forEach((dataset) => this._createNodesForDataset(dataset));
+    this._createCanvas();
```

Comment:
These last 3 lines are the cruz of what might be what calvin dislikes about this approach. It feels weird to me that we have one "renderAread" object created from svg group inside of "content", which may or may not be used for rendering. Then, we create a canvas which may or may not be used for rendering. I feel like this should be better abstracted and that a plot should have a renderer that will choose to attach one or the other itself. The plot won't know whether or not the renderer is using and SVG or canvas.